snuba/settings/validation.py

```diff
@@ -72,6 +73,7 @@ def validate_settings(locals: Mapping[str, Any]) -> None:
         "snuba-dead-letter-metrics-counters",
         "snuba-dead-letter-generic-events",
         "snuba-dead-letter-querylog",
+        "snuba-dead-letter-group-attributes",
```

Comment:
Ensure all of your topics exist in production - and the ops repo is updated with the topics in [prod_settings.py](https://github.com/getsentry/ops/blob/master/k8s/services/snuba/templates/prod_settings.py#L244). Without this, Snuba will fail to deploy due to topics mismatch.